docs/src/users_guide.md

```diff
@@ -72,89 +72,67 @@ instructions work.
-#   Hypergraph used : false
+#     1 : c min degree heuristic
+#     2 : c status 3 1618911763948
+#     3 : c min shortcut heuristic
+#     4 : c run with 0.0/0.1/0.2 min balance and node_min_expansion in endless loop with varying seed
+#   Hypergraph used : true
 #   Hyperedge contraction method : Netcon where possible, min fill heuristic otherwise.
 # Slicing :
 #   Method used : greedy treewidth deletion
 #   Edges sliced : 2
 #   Score fucntion used : direct_treewidth
-#   Treewidths after slicing consecutive edges : [1, 1]
+#   Treewidths after slicing consecutive edges : [1, 0]
 outputs 2
 load t1 data_1
 load t2 data_2
 load t3 data_3
 load t4 data_4
 load t5 data_4
-view t2_$v1 t2 3 $v1
-del t2
-view t3_$v1 t3 1 $v1
-del t3
-view t1_$v1 t1 1 $v1
-del t1
-view $o1_$v1 $o1 1 $v1
-del $o1
-ncon I1 2 t3_$v1 2 $o1_$v1 2
-del t3_$v1
-del $o1_$v1
-ncon I2 1 t1_$v1 1,2 t4 2
-del t1_$v1
-del t4
-ncon I3 1,3 t2_$v1 1,2,3 t5 2
-del t2_$v1
-del t5
-ncon I4 2 I3 1,2 $o2 1
-del I3
-del $o2
-ncon t8 2 I1 2 I2 2
-del I1
-del I2
-ncon t9 0 t8 1 I4 1
-del t8
-del I4
-save t9 output
+view o1_s o1 1 v1
+view t3_s t3 1 v1
+view t2_s t2 3 v1
+view t1_s t1 1 v1
+view t5_s t5 1 v2
+view t2_s_s t2_s 2 v2
+ncon I1 2,3 t2_s_s 1,2,3 o2 1
+ncon I2 1 t1_s 1,2 t4 2
+ncon t8 1 o1_s 1 t3_s 1
+ncon t9 1,3 t8 1 t5_s 3
+ncon t10 1 t9 1,3 I1 3,1
+ncon t11 0 t10 1 I2 1
+save t11 output
 ```
 
 ### DSL Format and Instructions
 
 The DSL file is a regular ASCII text file with one instruction per line. Lines beginning with `#` are comments which are
-ignored (except for the first line which contains version information). The first line should have a version string
+ignored (except for the first line which contains version information). The first line has a version string
 which specifies the format version. Comments following the first line contain metadata about the methods used
-to determine the contraction plan used and which edges to slice. Symbols prefixed by a dollar sign indicate variables which are take different values
-for each iteration of the computation. An explanation of each instruction follows
+to determine the contraction plan used and which edges to slice.
 
 #### Outputs
 ```
 outputs 2
 ```
 
-This line indicates that the tensor network has two output tensors. This will create symbols for each of the possible values of
-these output tensors. For a quantum circuit, measurement is performed the computational basis which means that the possible values for the output
-tensors are the vectors `[1, 0]` and `[0, 1]` corresponding to the $| 0 \rangle$ and $| 1 \rangle$ states respectively. For two outputs the following
-symbols and mappings will be defined
-
-```
-o1_0 => [1, 0]
-o1_1 => [0, 1]
-o2_0 => [1, 0]
-o2_1 => [0, 1]
-```
+This line indicates that the tensor network has two output tensors. These tensors are represented in subsequent DSL
+commands by symbols in the format `o{i}` where `{i}` is the index of the given output. To contraction for
```

Comment:
Maybe change "To contraction for ... " to "To contract the tensor network with different values for the output tensors, the data these symbols point to need to be updated with the new tensors."